parser/grammar.y

```diff
@@ -824,6 +824,13 @@ expression
           Strings: $3,
         }
       }
+    | primary_expression '%' _OF_ string_set
+      {
+        $$ = &ast.PercentOf{
```

Comment:
One thing we do in the yara compiler is validate that if the percentage is defined that it is between 1 and 100 (inclusive). I'm not doing that here but I suspect I should.